fedireads/templates/snippets/shelve_button.html

```diff
@@ -1,29 +1,22 @@
 {% load fr_display %}
 {% if request.user.is_authenticated %}
-{% if not pulldown %}
 
+<div class="dropdown secondary">
```

Comment:
I think we can use the existing css classes for the user icon pulldown and this if you use `pulldown-container` instead of `dropdown`, `pulldown` for `dropdown-content` and put the buttons in the form in a `ul` / `li`

Reply:
I can make this change if you prefer, since I'm probably more familiar with the css

Reply:
I wouldn't mind having a go at it to get more familiar with the css. The style is bit different, but I think you're right that it should be possible to unify them.

Reply:
awesome, go for it 🙂